common/src/main/java/net/vyhub/config/VyHubConfiguration.java

```diff
@@ -36,6 +36,7 @@ public static Map<String, String> loadConfig() {
             config.put("advert_prefix", "[★] ");
             config.put("advert_interval", "180");
             config.put("locale", "en");
+            config.put("send_succes_annoucments", "true");
```

Comment:
send_success_announcements is not very descriptive and it contains a spelling mistake. I propose `group_changed_notifications`